app/components/work_packages/share/modal_body_component.sass

```
@@ -1,6 +1,22 @@
 .op-share-wp-modal-body
   &--user-row
     display: grid
-    grid-template-columns: 1fr auto auto
-    grid-template-areas: "user button remove"
-    grid-gap: 10px
+    grid-template-columns: 1fr
+    grid-template-areas: "user"
+    grid-column-gap: 10px
+
+    &-manageable
```

In the core, we try to follow a BEM notation for the classes. With that, modifier class should start with an underscore.
```suggestion
    &_manageable
```

---

Thank you. I tried to follow this https://getbem.com/introduction/ to understand BEM but I see there's a different internal convention

---

https://github.com/opf/openproject/pull/13988/commits/446aed1c467f707321484704d409ccb1d749c4f0